push/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM openjdk:17-jdk-slim
```

Comment:
do not need now, cause we build Dockerfile on the flight

Reply:
so remove this file

Reply:
Done